src/components/tables/cells/ChipStatus.tsx

```diff
@@ -0,0 +1,34 @@
+import { AlertColor, Chip, TableCell, Theme } from '@mui/material';
+import { outlinedColoredChipBackground } from 'context/Theme';
+import { useIntl } from 'react-intl';
+
+interface Props {
+    messageId: string;
+    color: AlertColor;
+}
+
+const getOutlineColor = (theme: Theme, color: AlertColor) =>
+    theme.palette.mode === 'dark'
+        ? theme.palette[color].main
+        : theme.palette[color].dark;
+
+function ChipStatus({ messageId, color }: Props) {
+    const intl = useIntl();
+
+    return (
+        <TableCell>
+            <Chip
+                component="span"
+                label={intl.formatMessage({ id: messageId })}
+                size="small"
+                variant="outlined"
+                sx={{
+                    borderColor: (theme) => getOutlineColor(theme, color),
+                    backgroundColor: outlinedColoredChipBackground[color],
+                }}
```

Comment:
What's driving customized styling?

Reply:
An outlined chip with a text and background color set by the `color` property is hard to read in dark mode and the color is hard to distinguish as _green_ in light mode since the dark variant would need to be used for readability; a filled chip with a text and background color set by the `color` property is hard to read in both color modes.

Reply:
I can look into it. Are you thinking of adjusting the styling of all outlined chips that have a non-default color?

Reply:
yeah